examples/workflow/object_detection_3d/README.md

```diff
@@ -95,15 +95,15 @@ Run a script using the `--help` flag for more information:
 
 ```
 $ poetry run python3 object_detection_3d/seed_test_suite.py --help
-usage: seed_test_suite.py [-h] [--test-suite TEST_SUITE] sample-file
+usage: seed_test_suite.py [-h] [--test_suite TEST_SUITE] sample_file
 
 positional arguments:
   sample_file           File containing test sample and ground truth data
 
 options:
   -h, --help            show this help message and exit
-  --test-suite TEST_SUITE
-                        Name of test suite
+  --test_suite TEST_SUITE
```

Comment:
Thoughts on kebab case vs snake case? They both get parsed to the same variable in python, but I lean towards kebab because you already have hyphens in the option.

Reply:
I prefer kebab, but biased towards what we already had. Will happily make this change

Reply:
kebab gets another vote here

Reply:
second, i hate typing `_` in cli cmd

Reply:
Fourth'd